.github/workflows/mac_installer.yaml

```diff
@@ -0,0 +1,35 @@
+on:
+  release:
+    types: [published]
+
+
+jobs:
+  build:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
```

Comment:
Why false ?

Reply:
originally I had one workflow for both mac and ubuntu, this is no longer the case. I ll remove it.